.github/workflows/format.yml

```diff
@@ -0,0 +1,26 @@
+name: format-check
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags: '*'
```

Comment:
I think we could skip this and only run the check on PRs:
```suggestion
```

Reply:
Said and done!